crypto/bn256/cloudflare/gfp6.go

```diff
@@ -204,7 +204,51 @@ func (e *gfP6) Invert(a *gfP6) *gfP6 {
 	t1.Mul(B, &a.x).MulXi(t1)
 	F.Add(F, t1)
 
-	F.Invert(F)
+	F.InvertVariableTime(F)
```

Comment:
Just double-checking here - is it correct, and not a typo, to call `..VariableTime` from this method (`..ConstantTime`) ?

Reply:
There was a mistake here. Now all constant times are without `..ConstantTime` suffix and are default. Instead there is a variable time `Marshal()` function that is used by the concrete implementation of the precompile in the `contracts.go`